subsys/net/lib/lwm2m_client_utils/Kconfig

```diff
@@ -59,6 +59,19 @@ config LWM2M_CLIENT_UTILS_DOWNLOADER_SEC_TAG
 
 endif # LWM2M_CLIENT_UTILS_FIRMWARE_UPDATE_OBJ_SUPPORT
 
+config LWM2M_CLIENT_UTILS_SIGNAL_MEAS_INFO_OBJ_SUPPORT
+	bool "Signal measurement information object support (ID 10256)"
+	default y
+	help
+	  This object is used in combination with Connectivity Monitor object (ID 4)
+	  and provides information about neighbouring cells. It is used to assist
+	  with the device location services.
```

Comment:
I think we are giving a way documentation that might not always be relevant.
We MAY use this for locationing purposes, however, it does not affect the code at all. Somebody else may use for some other purposes.
So should we document just what this functionality does here, which is providing the information regarding neighbor cells.